src/cursorless.talon

```diff
@@ -5,32 +5,26 @@ tag(): user.cursorless
 {user.cursorless_simple_action} <user.cursorless_target>:
     user.cursorless_simple_action(cursorless_simple_action, cursorless_target)
 
+{user.cursorless_multiple_target_action} <user.cursorless_multiple_targets>:
+    user.cursorless_multiple_target_command(cursorless_multiple_target_action, cursorless_multiple_targets)
+
 <user.cursorless_wrapper> wrap <user.cursorless_target>:
     user.cursorless_single_target_command_with_arg_list("wrap", cursorless_target, cursorless_wrapper)
 
-wrap <user.cursorless_target> with funk <user.code_functions>:
-    user.cursorless_single_target_command("wrap", cursorless_target, "{code_functions}(", ")")
+# wrap <user.cursorless_target> with funk <user.code_functions>:
+#     user.cursorless_single_target_command("wrap", cursorless_target, "{code_functions}(", ")")
 
 replace <user.cursorless_target> with <user.cursorless_replace_value>$:
     user.cursorless_replace(cursorless_target, cursorless_replace_value)
 
-reformat <user.cursorless_target> as <user.formatters>:
+format <user.formatters> onto <user.cursorless_target>:
```

Comment:
ok so this one is pretty awkward, but surprisingly swapping these two captures seems to improve DFA compilation.  We could prob come up with better terminology here

Reply:
Maybe we should just remove onto and make it `format camel blue air`